app/views/spree/admin/shared/_subscription_tab.html.erb

```diff
@@ -1,3 +1,3 @@
 <% if can? :admin, SolidusSubscriptions::Subscription %>
-  <%= tab :subscriptions, icon: 'repeat' %>
+  <%= tab :subscriptions, icon: 'repeat', match_path: '/subscriptions' %>
```

Comment:
Is this related to this PR? If yes, I'd use a separate commit that explains how they are related, otherwise a new PR.

Reply:
This is necessary to avoid having multiple options selected in the menu. I left it in the same PR to avoid having a known bug in a revision that could be visited in the future.

![image](https://user-images.githubusercontent.com/2119750/97719070-6d438680-1aa5-11eb-8dd6-3d5de7c251a6.png)

Reply:
I think a separate commit would be ideal for that change, but this is not blocking.